src/components/book.js

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { deleteBook } from '../redux/api/api';
+
+const Book = ({ books }) => {
+  const dispatch = useDispatch();
+
+  const getBookId = (e) => {
+    const btnId = e.target.id;
+    deleteBook(btnId, dispatch);
+  };
```

Comment:
- [ ] I guess that this function is meant to delete a book, its name is not really meaningful I would suggest renaming it to something more clear like `handleRemoveBook`.
- [ ] You already have the `book.item_id` on your book object so you don't need to take it from the `event` object. To make sure that you pass it as an argument to your function you need to use a callback function in the button `onClick` handler. 
```
 <button type="button" id={book.item_id} onClick={() => handleRemoveBook(book.item_id)}>
```